### DISCUSSION TITLE
Automatic Dark Mode / Use system preferences

### DISCUSSION THREAD

**Discussion by tooomm** [iotaledger/firefly#1579] [Features / Enhancements]
## Description

Currently, there is `Light theme` and `Dark theme` support.
There is no ability to swap automatically between them or use/synchronize with your system default.

## Motivation

- One have to decide between the both, while there is clearly no right and wrong and preference is dependant on external factors like time or environment. For a great app experience the user has to switch regularly.
- Manual changing is tedious via the settings.
- There are smarter possibilities available in basically all modern operating systems to manage night shift, themes and particularly "Dark Mode".

## Requirements

1. Introduce ~`Use System Default` option next to strict `Dark` and `Light`
2. This should use the system preference and make the app switch between the two options instead trying to manage it manually and pin a certain theme.

## Are you planning to do it yourself in a pull request?

No.

**Comment by maxwellmattryan** [comment]
Hey @tooomm, thanks for your suggestion. I actually had made changes adding this into the settings, but it was before Ledger ramped up really so it's a little behind now 😅. I am going to close this issue and move it to a discussion [here](https://github.com/iotaledger/firefly/discussions/1507) 👍

**Comment by cvarley100** [comment]
This has since been implemented.
